### PR TITLE
Skip yaml file validation

### PR DESCRIPTION
**Pull request by hejingkan2005**
- validate .md file only

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/5277)

### CODE REVIEW

**Comment by yufeih** [comment]
Can you add a spec test that covers:

1. yaml files are not validated
2. markdown files included by yaml files are validated

**Comment by hejingkan2005** [comment]
> Can you add a spec test that covers:
> 
> 1. yaml files are not validated
> 2. markdown files included by yaml files are validated

Sure, done.

File: docs/specs/validation.yml [hunk 1]
```diff
@@ -340,3 +340,74 @@ outputs:
...
+    {
+      "headings": {
+        "name": "Headings",
+        "description": "Validates H1 and other heading content",
+        "aliases": null,
+        "rules": [
+          {
+            "type": "H1Missing",
+            "message": "H1 is required. Use a single hash (#) followed by a space to create your top-level heading.",
+            "exclusions": [ "toc", "includes", "hubpage", "landingpage" ],
```
Comment (yufeih) [inline docs/specs/validation.yml]: Do we want respect this `exclusions` property or blindly exclude all markdown files? If there is no intend to support this property, consider removing it from the test case, as it brings confusion.

Comment (hejingkan2005) [reply docs/specs/validation.yml]: > Do we want respect this `exclusions` property or blindly exclude all markdown files? If there is no intend to support this property, consider removing it from the test case, as it brings confusion.

Done.

**Pull request merged**